src/Cake.Common/Tools/OpenCover/OpenCoverRunner.cs

```diff
@@ -165,6 +165,13 @@ public sealed class OpenCoverRunner : Tool<OpenCoverSettings>
                 builder.AppendSwitch("-log", ":", settings.LogLevel.ToString());
             }
 
+            // HideSkipped Option
+            if (settings.HideSkippedOption != OpenCoverHideSkippedOption.All)
```

Comment:
I think the default is empty, so nothing gets skipped. You can then define the things that should be skipped and all seems to be just all of them. So we probably need a `HideSkippedOption.None` or `HideSkippedOption.Default` which you can use here to check.

Reply:
Added `None` option.